### DISCUSSION TITLE
Love to participate

### DISCUSSION THREAD

**Discussion by The-Young-Programmer** [Samuel-de-Oliveira/Helen#2] [General]
Can you elaborate more on the project?

**Comment by Samuel-de-Oliveira** [comment]
## Hello!

This project is a conclusion homework of my course (But I want to proceed with it anyway to help people)
 
The project is a Repository in Github with arduino code and tutorial to create a equipment to help people with vision disability (Blinded, low vision and deaf blind) know if has something in them front.

And it's will be universal, this means that can be used in anywhere (Like clothes, hats or in the blind cane) and have various versions of the project (With a single distance sensor, with two distance sensors and others).

So basically the project is not done yet, then I want to finish first the tutorial and some code versions to create the first release.

that's all :+1:

**Comment by The-Young-Programmer** [comment]
Okay, this is a very nice project. I'm interested in this project and I love to develop/work with you.

According to my understanding, first thing first is to get the instrument or tools you will be needing for the project , tools like :

1. Arduino UNO Board
2. Pixy camera board
3. Sensors, like:

- proximity sensors
- ultrasonic sensors
- capacitive and photoelectric sensors

Factors to also consider:

1. Range – This involves choosing the sensors based on measurement limits.
2. Environmental Condition – These include sensor operational limits like temperature and humidity.
3. Device container

**Comment by Samuel-de-Oliveira** [comment]
I've read your post and I noticed somethings.

### 1. The pixy camera

Is a good idea but I want to make it more simple, because I think that sensors should be more convenient.

### 2. The Range limit, Environmental condition and capacitive sensor
#### Range limit:
Well, I had thought about it. in the [Single ultrasonic code](https://github.com/Samuel-de-Oliveira/Helen/blob/main/Helen_Single_HC-SR04/Helen_Single_HC-SR04.ino) Have a part with the title *constants* and you can see that have the limit distance of beeps with the measure (everything you can change before upload to arduino). Take a look in the code to understand it better.

#### Environmental condition:
I don't think about it, probably I will add this kind in the project

#### Capacitive Sensor:
This should be cool to implement in the project. I will try add this

### 3. Device container
for a while i'm trying to create one, but I'm focusing in the code and sensor working.
But i accept ideas for.

### 4. Yes, I'm accepting help
Thats okay, I accept your help.

**Comment by The-Young-Programmer** [comment]
- I think that it's a great idea to use sensors. I recommend using (HC-SR05)
- I'm going to take a look at the code and see if I can make a pull request.

**Have you run test this <a href="https://github.com/Samuel-de-Oliveira/Helen/blob/main/Helen_Single_HC-SR04/Helen_Single_HC-SR04.ino">Helen_Single_HC-SR04</a>**
